cmake/caches/windows-x86_64-swift-flags.cmake

```diff
@@ -1,9 +1,12 @@
 set(CMAKE_Swift_COMPILER_TARGET x86_64-unknown-windows-msvc CACHE STRING "")
-set(CMAKE_Swift_FLAGS "-resource-dir \"${CMAKE_Swift_SDK}/usr/lib/swift\" -L${CMAKE_Swift_SDK}/usr/lib/swift/windows" CACHE STRING "")
-set(CMAKE_Swift_LINK_FLAGS "-resource-dir \"${CMAKE_Swift_SDK}/usr/lib/swift\" -L${CMAKE_Swift_SDK}/usr/lib/swift/windows" CACHE STRING "")
 
-set(CMAKE_SWIFT_FLAGS "-resource-dir \"${CMAKE_Swift_SDK}/usr/lib/swift\"" CACHE STRING "")
-set(CMAKE_SWIFT_LINK_FLAGS "-resource-dir \"${CMAKE_Swift_SDK}/usr/lib/swift\"" CACHE STRING "")
+set(SWIFT_STDLIB_DIR "${CMAKE_Swift_SDK}/usr" CACHE STRING "")
```

Comment:
I suppose that this is okay ... since it computes it based on the variable that we want users to use longer-term.

Reply:
Not sure about naming though. Is it ok as is? `CMAKE_Swift_STDLIB` better follows general style, but it is not really related to CMake Swift support.

Reply:
Yeah, I expect to make `CMAKE_Swift_SDK` a proper CMake variable, but the handling here is special, and so not using the CMake spelling is fine I think.